### PR TITLE
Add multi types

### PR DESCRIPTION
**Pull request by yuhui1208**

### CODE REVIEW

File: packages/type-definitions/src/index.ts [hunk 1]
```diff
@@ -10,8 +10,11 @@ import locks from './locks';
 import market from './market';
 import staking from './staking';
 import swork from './swork';
+import maxwellBenefit from './maxwellBenefit';
+import maxwellMarket from './maxwellMarket';
+import maxwellSwork from './maxwellSwork';
 
-export const crustTypes = {
+export const mainnet = {
```
Comment (zikunfan) [inline packages/type-definitions/src/index.ts:17]: Change to `crust`

File: packages/type-definitions/src/index.ts [hunk 2]
```diff
@@ -21,29 +24,59 @@ export const crustTypes = {
...
+export const maxwell = {
+  base,
+  claims,
+  maxwellBenefit,
+  maxwellMarket,
+  maxwellSwork,
+  staking,
+};
+
+export const mainnetTypes = {
```
Comment (zikunfan) [inline packages/type-definitions/src/index.ts:36]: `crustTypes`

**Review by zikunfan — CHANGES_REQUESTED** [review]

File: packages/type-definitions/src/index.ts (hunk 1 above)
Comment (yuhui1208) [reply packages/type-definitions/src/index.ts]: fixed

File: packages/type-definitions/src/index.ts (hunk 2 above)
Comment (yuhui1208) [reply packages/type-definitions/src/index.ts]: fixed

**Pull request closed without merging**